src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -652,11 +652,6 @@ private void printRankings(
       throws IOException {
     // for each rank determine what candidate id, overvote, or undervote occurred and print it
     for (int rank = 1; rank <= maxRanks; rank++) {
-      // If the configuration did not allow this ranking, we want to exclude it
-      // from the RCTab CVR -- even if it was present in the source CVRs.
-      if (!reader.isRankingAllowed(rank, source.getContestId())) {
-        break;
-      }
```

Comment:
Is the idea that this is no longer needed because it'll hit a halting exception before it has a chance to get to this state?

Reply:
Yeah, I think this is now supposed to get caught when we first read in the CVRs.